index.html

```diff
@@ -1,11 +1,144 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
-    <meta http-equiv="refresh" content="0;url=https://github.com/seriouslysean/monster-hunter-now-events">
-    <title>Redirecting...</title>
+    <title>Monster Hunter Events Today</title>
 </head>
-<body>
-    If you are not redirected, <a href="https://github.com/seriouslysean/monster-hunter-now-events">click here</a>.
+
+<body style="
+        background-color:#F6F4E4;
+        color: #222;
+        padding: 0px;
+        margin: auto;
+        display: flex;
+        flex-direction:column;
+        row-gap: 15px;
+        align-items: center;
+        text-align: center;
+        overflow-x: hidden;">
+    <!-- Page Heading -->
+    <div style="
+        font-size: 60px;
+        font-style: italic;
+        font-weight: 700;
+    ">Monster Hunter Events Today</div>
+
+    <!-- Date -->
+    <div style="
+        font-size: 40px;
+        font-style: italic;
+        font-weight: 700;" id="tag_display_date">
+
+    </div>
+
+    <!-- Display Event Cards Here -->
+    <div style="
+        display:flex;
+        flex-direction:column;
+        gap:20px;" id="tag_display_events">
+    </div>
+
+    <script type="text/javascript">
+
+        let eventsTodayList = [];
+        const TODAY = new Date();
+
+        function appendData(data) {
+            data.events.forEach((event) => {
+                // console.log("event: ", event);
+                let flag = 0;
+                event.dates.forEach((date) => {
+                    // console.log("date: ", date);
+                    if (new Date(date.start) <= TODAY && new Date(date.end) >= TODAY) {
+                        flag = 1;
+                    }
+                })
+                if (flag == 1) {
+                    // The Event Occurs Today
+                    eventsTodayList.push({
+                        name: event.summary,
+                        desc: event.description,
+                    });
+                }
+            });
+
+            return eventsTodayList;
+        }
+
+
+        fetch('/json')
```

Comment:
```suggestion
        fetch('/dist/events.json')
```
Prefer the actual hosted dist file so we get the generated and deduped content.

Reply:
Actually, this route (/json) is defined in the server.js file, where the original file is hosted from /dist/events.json
So the generaetd content is directly accessed from the original location anyway...

Reply:
Right, but it’s still an alias to the file which is not preferable imo.